Makefile

```diff
@@ -50,8 +51,8 @@ release:
 	@echo ">> building release binaries"
 	@./build/release.sh
 
-docker:
-	@docker build -t cadvisor:$(shell git rev-parse --short HEAD) -f deploy/Dockerfile .
+docker-%:
+	@docker build -t cadvisor:$(shell git rev-parse --short HEAD) -f deploy/Dockerfile-$* .
```

Comment:
Does this mean we need to rename the other Dockerfile to be arch-specific?

Reply:
We would have architecture specific Dockerfiles provided, to enable the build of the cadvisor binary in that respective arch environment.
I have changed the logic, such that we wouldn't need to update the makefile everytime, for each  new architecture added.